src/test/java/org/openelisglobal/unitTests/organization/OrganizationContactIntegrationTest.java

```diff
@@ -0,0 +1,46 @@
+package org.openelisglobal.unitTests.organization;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+
+
+
+import org.openelisglobal.organization.valueholder.Organization;
+import org.openelisglobal.organization.valueholder.OrganizationContact;
+import  org.openelisglobal.person.valueholder.Person;
+
+
+
+
+public class OrganizationContactIntegrationTest {
+    private Person person;
+    private Organization organization;
+
+    @Before
+    public void setUp() {
+        // Initialize Person
+        person = new Person();
+        person.setFirstName("John");
+        person.setLastName("Doe");
+
+        // Initialize Organization
+        organization = new Organization();
+        organization.setId("123");
+    }
+
+    @Test
+    public void testOrganizationContactIntegration() {
```

Comment:
Thanks @sayed4900 for attemtoing this. 
For consistence , 
test methods should follow a naming convetion like 
`MethodName_ExpectedBehavior` ie _getOrganization_shouldReturnOrganization_